help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,245 @@
+
+## Connect to the destination
+
+[!IMPORTANT]
+
+To connect to the destination, you need the **View
+Destinations** and **Manage and Activate Dataset Destinations** [[access
+control
+permissions]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions).
+Read the [[access control
+overview]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or
+contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in
+the [[destination configuration
+tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination).
+In the destination configuration workflow, fill in the fields listed in
+the two sections below.
+
+### Authenticate to destination
+
+To authenticate to the destination, fill in the required fields and
+select **Connect to destination**.
+
+To access your bucket on Experience Platform, you need to provide valid
+values for the following credentials:
+
+  ------------------------------------------------------------------------------
+  **Credential**   **Description**
+  ---------------- -------------------------------------------------------------
+  Access key       The access key ID for your bucket. You can retrieve this
+                   value from the Merkury team.
+
+  Secret key       The secret key ID for your bucket. You can retrieve this
+                   value from the Merkury team.
+
+  Bucket name      This is your bucket where files will be shared. You can
+                   retrieve this value from the Merkury team.
+  ------------------------------------------------------------------------------
+
+### New Account
+
+To define a new Merkury Managed S3 location:
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image4.png){width="6.5in"
+height="3.509027777777778in"}
+
+### Existing Account
+
+Accounts already defined using the Merkury Enterprise Connections
+destination appear in a list pop-up. When selected, you can see details
+on the account in the right rail. View the example from the UI, when you
+navigate to **Destinations** \> **Accounts**;
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image5.png){width="7.260470253718285in"
+height="1.5785312773403324in"}
+
+### **Fill in destination details
+
+To configure details for the destination, fill in the required and
+optional fields below. An asterisk next to a field in the UI indicates
+that the field is required.
+
+![A screenshot of a computer Description automatically
+generated](./merkury-connections/media/image6.png){width="6.5in"
+height="3.877083333333333in"}
+
+-   **Name (Required)** - The name the destination will be saved under
```

Comment:
formatting, please keep the formatting from the template

Reply:
Addressed